common/aws/ecs/ecs.go

```diff
@@ -769,8 +769,8 @@ func (this *ECS) ListClusterServiceNames(clusterName, prefix string) ([]string,
 	fn := func(output *ecs.ListServicesOutput, lastPage bool) bool {
 		for _, serviceARN := range output.ServiceArns {
 			// sample service ARN:
-			// arn:aws:ecs:us-west-2:856306994068:service/l0-tlakedev-guestbo80d9d
-			serviceName := strings.Split(aws.StringValue(serviceARN), "/")[1]
```

Comment:
Any particular reason why we made this only forwards compatible? Something like the below should've handled both new and old formats. Primary concern is that this is one additional thing we have to remember to check if scaling isn't working correctly (i.e., whether the layer0 user or account is still opted in).

```golang
temp := strings.Split(aws.StringValue(serviceARN), "/")
serviceName := temp[len(temp)-1]
```